### PR TITLE
Update FAQ entry about contributing

### PR DESCRIPTION
**Pull request by nmattia**

### CODE REVIEW

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

File: src/frontend/src/flows/faq/index.ts [hunk 1]
```diff
@@ -132,9 +132,17 @@ export const questions = {
...
+        If you'd like to contribute to Internet Identity, please head over to
+        the
+        ${mkExternalLink({
+          href: "https://github.com/dfinity/internet-identity",
+          text: "GitHub repository",
+        })}
+        and check out the
+        ${mkExternalLink({
+          href: "https://github.com/dfinity/internet-identity/blob/main/CONTRIBUTING.md",
+          text: "contributing guidelines",
```
[the remark this answers is not in the document]
Comment (nmattia) [reply src/frontend/src/flows/faq/index.ts:144]: Mh I think "guide" is a bit misleading don't you think? it's not like we're "guiding" or hand holding...

[the remark this answers is not in the document]
Comment (peterpeterparker) [reply src/frontend/src/flows/faq/index.ts:144]: Don't mind me, was just a suggestion. PR already accepted.

[the remark this answers is not in the document]
Comment (nmattia) [reply src/frontend/src/flows/faq/index.ts:144]: This is also the "official" name [used by GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), so will stick with "guidelines". Thanks for the suggestion though!

**Pull request merged**